src/main/java/com/fundation/search/view/SearchPanel.java

```diff
@@ -106,4 +287,37 @@ public JButton getButtonSearch() {
     public JTable getTableResults() {
         return tableResults;
     }
+
+    public String getOptionSize() {
+        return optionSize.getSelectedItem().toString();
+    }
+
+    public String getSizeTextField() {
+        return size.getText();
+    }
+
+    public String getExtension() {
+        return extension.getText();
+    }
+
+    public boolean getReadOnly() {
```

Comment:
Please change the return for **Boolean**, due that **boolean** won't return a null for a non selected option